startup-docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -eu
+
+export RUN_TESTS=${RUN_TESTS:-false}
+export USE_LOCAL_BUILD=${USE_LOCAL_BUILD:-false}
+export VERIFY_USE_PUBLIC_BINARIES=${VERIFY_USE_PUBLIC_BINARIES:-false}
+
+BUILD_IMAGES=${1:-false}
+
+if [ $BUILD_IMAGES = "--build" ]; then
+  echo "Building images..."
+  docker-compose build --parallel
+fi
+
+echo "Starting Gateway, Translator, Stub Connector, VSP..."
+docker-compose up -d stub-connector proxy-node-gateway translator verify-service-provider
+
+# ESP won't start without Connector metadata
+printf "Waiting for Stub Connector"
+until $(curl --output /dev/null --silent --head --fail http://localhost:6610/ConnectorMetadata); do
+  printf '.'
+  sleep 2
+done
+echo
```

Comment:
Is this redundant?

Reply:
Nope - because above we use `printf` so there's no newline at the end